### PR TITLE
Product-service_CU-jfaz7a

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-jfaz7a Product service](https://app.clickup.com/t/jfaz7a)

**Comment by lpboucher** [comment]
Task linked: [CU-jfaz7a Product service](https://app.clickup.com/t/jfaz7a)

File: src/app/core/products/product.service.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            type: "coffee"
+        },
+    ];
+
+
+    constructor() { }
+
+    ngOnInit() {
+    }
+
```
Comment (lpboucher) [inline src/app/core/products/product.service.ts]: ngOnInit should not be used.

Services are not part on the render cycle and so lifecycle methods do not apply

File: src/app/modules/products/components/product-list/product-list.component.ts [hunk 2]
```diff
@@ -1,111 +1,22 @@
 import { Component, OnInit } from '@angular/core';
-import { Observable, of } from "rxjs";
+import { Observable } from 'rxjs';
 
-import { Product } from "@app/shared/models/classes/product.interface";
-import { Coffee } from "@app/shared/models/classes/coffee.interface";
+import { Coffee } from '@shared/models/classes/coffee.interface';
+
+import { ProductService } from '@app/core/products/product.service';
```
Comment (lpboucher) [inline src/app/modules/products/components/product-list/product-list.component.ts]: here, we have an alias path for the core module, so we can import as:
"@core/products/product.service"

you can see all aliases in the file tsconfig.json

File: src/app/core/products/product.service.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from "rxjs";
+
+import { Coffee } from "@app/shared/models/classes/coffee.interface";
+
```
Comment (lpboucher) [inline src/app/core/products/product.service.ts:6]: see comment below, can be imported from @shared

**Review by lpboucher — CHANGES_REQUESTED** [review]
simple changes to be made. Make sure to pull before making further changes, I adjusted double quotes

**Review by lpboucher — APPROVED** [review]

**Pull request merged**